static/src/javascripts/projects/common/modules/loyalty/save-for-later.js

```diff
@@ -10,8 +10,8 @@ define([
     'common/utils/template',
     'common/modules/identity/api',
     'common/views/svgs',
-    'text!common/views/loyalty/save-for-later--signed-out.html',
-    'text!common/views/loyalty/save-for-later--signed-in.html'
+    'text!common/views/loyalty/save-for-later-link.html',
+    'text!common/views/loyalty/save-for-later-button.html'
```

Comment:
| Saved | Unsaved
--- | --- | ---
Logged in | link to SFL page | button and XHR
Not logged in | link to login -> SFL page | N/A